### PR TITLE
Request and Response can include array

### PR DESCRIPTION
**Pull request by sinanmutlu**
Request and Response can include array

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@Necrozma Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-guides/gs-producing-web-service&pullRequestId=12)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-guides/gs-producing-web-service&pullRequestId=12) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@Necrozma Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-guides/gs-producing-web-service&pullRequestId=12)!

File: complete/request.xml [hunk 1]
```diff
@@ -4,6 +4,7 @@
    <soapenv:Body>
       <gs:getCountryRequest>
          <gs:name>Spain</gs:name>
+		 <gs:name>Poland</gs:name>
```
Comment (dsyer) [inline complete/request.xml]: Tabs instead of spaces?

File: complete/pom.xml [hunk 2]
```diff
@@ -14,7 +14,7 @@
 	</parent>
 
     <properties>
-        <java.version>1.8</java.version>
+        <java.version>1.7</java.version>
```
Comment (dsyer) [inline complete/pom.xml]: Why?

**Comment by dsyer** [comment]
You might want to add a bit of detail to your description of this PR, so we can understand why you want to make the change. It doesn't really seem to add much to me.

**Comment by Necrozma** [comment]
In order to request more than one object and also for multiple responses i have changed the xsd file and accordingly reorganized endpoint file. Tabs are not important changes, i used it because it looks good soap ui. Java 1.7 was my default jdk.

**Comment by dsyer** [comment]
It's not really a material change then. I'm happy to leave the PR open for other people to copy, but I don't see much value in merging it TBH. (And Java 8 is the baseline for all guides BTW.)

**Comment by gregturn** [comment]
Thanks @sinanmutlu but I think we're going to decline your requested changes.

**Pull request closed without merging**